azure_functions_worker/mmap_handler/file_accessor.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import abc
+import mmap
+from typing import Optional
+
+
+class FileAccessor(metaclass=abc.ABCMeta):
+    """
+    For accessing memory maps.
+    This is an interface that must be implemented by sub-classes to provide platform-specific
+    support for accessing memory maps.
+    Currently the following two sub-classes are implemented:
+        1) FileAccessorWindows
```

Comment:
Windows is not a file right?

Reply:
Yes, will rename to `SharedMemoryMapAccessor*`.
I am also thinking of refactoring this similar to what we have in C# i.e. have a class called `SharedMemoryMap` and do all the operations which are being done in `FileReader`/`FileWriter` as part of that class.